src/vivarium_csu_alzheimers/components/testing.py

```diff
@@ -11,15 +13,25 @@
 NEGATIVE_TEST_TRANSITION_RATE = TESTING_RATE * (1 - POSITIVE_TEST_RATE)
 
 
+class TestingState(TransientState):
+    """A transient state that will be compatible with the vivarium public health disease model."""
+
+    def get_transition_names(self) -> list[str]:
```

Comment:
This was the thing we discussed should possibly exist in vivarium's `Transient` class, right?

Reply:
Yes